assets/blocks/quiz/quiz-block/use-has-questions.js

```diff
@@ -16,28 +16,39 @@ export const useHasQuestions = ( clientId ) => {
 			.filter( ( block ) => !! block.attributes.title )
 	);
 
+	const { editPost } = useDispatch( 'core/editor' );
+	const setQuizHasQuestions = useCallback(
+		( on ) => {
```

Comment:
Would `hasQuestions` make more sense than `on` here?

Reply:
Renamed to `enable`, that seems to be a common toggle parameter name.